components/server/src/gitlab/api.ts

```diff
@@ -256,19 +255,19 @@ export namespace GitLab {
     export namespace Permissions {
         export function hasWriteAccess(repo: Project): boolean {
             if (repo.permissions.project_access) {
-                return repo.permissions.project_access.access_level >= 30;
+                return (repo.permissions.project_access as unknown as { access_level: number }).access_level >= 30;
```

Comment:
Is no proper type export anymore? This looks like we can stop use typing at all. Let me double-check.